scrapers/Fuckermate.yml

```
@@ -0,0 +1,34 @@
+name: Fuckermate
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fuckermate.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="post-header font-alt"][1]/h1()
+      Details: 
+        selector: //*[@id="video-section"]/div/div/div/div/div[3]/div/div[1]/p[2]
+      Image:
+        selector: //div[@class="vjs-poster"]/@style
+        postProcess:
+          - replace:
+              - regex: .+(?:url\(")(.+)(?:"\)\;)
+                with: $1
```

```suggestion
      Image: //videoplayer/@poster
```
this works and doesnt need the cdp